app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,23 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/welcomeText"
```

Comment:
please follow the naming convention. The XML IDs shoud be in sneake_case. In this case should be welcome_text

Reply:
done